h2o-core/src/main/java/water/api/Schema.java

```diff
@@ -341,6 +341,21 @@ public S fillFromParms(Properties parms, boolean checkRequiredFields) {
     return (S) this;
   }
 
+  /**
+   * Fills this Schema from the body content when available.
+   * By default the body is interpreted as JSON object.
+   *
+   * We use PojoUtils.fillFromJson() rather than just using "schema = Gson.fromJson(post_body)"
+   * so that we have defaults: we only overwrite fields that the client has specified.
+   *
+   * @param body the post body (can't be null), converted to JSON by default
+   * @return the filled schema
+   */
+  public S fillFromBody(String body) {
+    PojoUtils.fillFromJson(this, body);
+    return (S) this;
+  }
```

Comment:
rationale for this new method:
- provide a simple way to customize parsing of body params: necessary in AutoML.
- method name symmetric with `fillFromParams`